### PR TITLE
adding tricks section

### PR DESCRIPTION
**Pull request by sh6khan**

### CODE REVIEW

**Comment by rperryng** [comment]
actually didn't know about most of these. :shipit:

File: README.md [hunk 1]
```diff
@@ -277,6 +283,76 @@ _Why Rebase?_, It's a great to make a clean looking tree, it appears as if you n
...
+  ```bash
+  $ git config --global alias.alias_command git_command
+  ```
+
+Examples:
+
+- Set `git stat` to `git status -sb`
+
+  ```bash
+  $ git config --global alias.stat status -sb
```
Comment (rperryng) [inline README.md:353]: I'm pretty sure you have to wrap `status -sb` in quotations (" ") in order for it to group the option in with the alias

Comment (sh6khan) [inline README.md:353]: right, you do 
mah bad

**Comment by rperryng** [comment]
:ship:

**Pull request merged**